### ISSUE TITLE
Unable to download Spotify from xManager

### ISSUE THREAD

**Issue by Sarvagon** [xManager-v2/xManager-Spotify#112]
**Describe the bug**
Since the last update of xManager (3.4), it's impossible to display the available versions of Spotify (nothing appears when clicking on the button to display versions)

**To Reproduce**
1. Go to 'xManager' app
2. Click on 'Versions'
3. See error

**Expected behavior**
When clicking on the button, versions available should appear.

**Screenshots**
![Screenshot_20220427-183053](https://user-images.githubusercontent.com/63664894/165568099-4444ff5e-a85e-46ba-8dec-d16e70069cd7.png)

**Smartphone (please complete the following information):**
 - Device: [Sony Ericsson Xperia XA1]
 - OS: [e.g. Android Oreo (8.0.0)]
 - Version [G3121]

**Additional context**
I did several tests on different android versions (8.0 and later) and it seems that it definitely don't work for Oreo version.

**Comment by xC3FFF0E** [comment]
Kindly update to v3.5. Let me know if the issue persists. Thanks!

**Comment by Sarvagon** [comment]
> Kindly update to v3.5. Let me know if the issue persists. Thanks!

Hi, there is still the issue with version 3.5.
Thanks for your reply :)

**Comment by Sarvagon** [comment]
> Kindly update to v3.5. Let me know if the issue persists. Thanks!

Just to let you know, the issue is also in Android 12 (Sony Xperia 1 III [XQ-BC72]). The only temporary solution I found is to reinstall Xmanager. However, after a few moment, the problem reappears.

**Comment by djapexlo** [comment]
> > Kindly update to v3.5. Let me know if the issue persists. Thanks!
> 
> Just to let you know, the issue is also in Android 12 (Sony Xperia 1 III [XQ-BC72]). The only temporary solution I found is to reinstall Xmanager. However, after a few moment, the problem reappears.

Yeah, it still happens in 3.6 version. Im on Android 10 (Xiaomi Mi A2)
If someone can link me url of just spotify app that would be apreciated :)

**Comment by Sarvagon** [comment]
> > > Kindly update to v3.5. Let me know if the issue persists. Thanks!
> > 
> > 
> > Just to let you know, the issue is also in Android 12 (Sony Xperia 1 III [XQ-BC72]). The only temporary solution I found is to reinstall Xmanager. However, after a few moment, the problem reappears.
> 
> Yeah, it still happens in 3.6 version. Im on Android 10 (Xiaomi Mi A2) If someone can link me url of just spotify app that would be apreciated :)

Yes you're right, I tried xManager v3.6 on my Sony Xperia 10 (Android 10) and the problem still happens. It's strange because with lastest versions I never had this problem.
